src/hotspot/share/runtime/globals.hpp

```diff
@@ -2088,22 +2091,24 @@ const intx ObjectAlignmentInBytes = 8;
   develop(bool, TraceOptimizedUpcallStubs, false,                           \
                 "Trace optimized upcall stub generation")                   \
                                                                             \
-  product(ccstr, CRaCCheckpointTo, NULL, "Path to checkpoint image")        \
+  product(ccstr, CRaCCheckpointTo, NULL, MANAGEABLE,                        \
```

Comment:
I see reasons for CRaCCheckpointTo to be MANAGEABLE, but at the moment the flag is assumed to be set in the command line by the implementation, e.g. os::Linux::{vm_create_start,prepare_checkpoint} are called depending on the flag value, and that can happen only during VM initialization.

A set of changes are required before the option can become MANAGEABLE.

The test should also updated once the option is reverted.

Reply:
Well spotted, I was thinking about changing the path but these two places need to be called in order to prepare for a checkpoint.